apps/game_backend/lib/game_backend/users/user.ex

```diff
@@ -47,6 +47,10 @@ defmodule GameBackend.Users.User do
     has_many(:currency_caps, UserCurrencyCap)
 
     timestamps()
+
+    # Virtual fields for client display
+    field(:prestige, :integer, virtual: true)
+    field(:quest_refresh_at, :any, virtual: true)
```

Comment:
Why are this virtual? Don't we want to store them?

Reply:
Actually, you don't want to store this field. This is just to send the refresh date (a simple utc.now) to the client so all stay consistent. Maybe for some reason the server and the client have a different hour and that will cause inconsistency, the field that you wanna persist is `last_daily_quest_generation_at`